docs/introduction/getting-started.stories.mdx

```diff
@@ -61,11 +61,11 @@ export default function App() {
 
 ### Configuring the viewport
 
-Finally, make sure that the viewport meta tag includes `viewport-fit=cover`, and that your app has the right padding to render inside [CSS safe areas](https://developer.mozilla.org/en-US/docs/Web/CSS/env()):
+Finally, make sure that the viewport meta tag includes `viewport-fit=cover`, and that your app has the right padding to render inside [CSS safe areas](<https://developer.mozilla.org/en-US/docs/Web/CSS/env()>):
```

Comment:
My linter did this, I guess we can leave it (?)

Reply:
What, why? That doesn't look valid. Did you test if it still works?

Reply:
(in the URL)

Reply:
Probably a way to "escape" weird URL characters